lib/filters/handlerMonitor.js

```diff
@@ -24,9 +24,10 @@ Filter.prototype.after = function (err, msg, session, resp, next) {
   const duration = Date.now() - msg.__statsd__.startTime
   const tags = {
     route: msg.__route__,
-    success: !err,
+    error: err,
+    type: 'http'
   }
 
-  this.client.histogram('pomelo.handler.response_time', duration, tags)
+  this.client.timing('pomelo.response_time_ms', duration, tags)
```

Comment:
since you removed the game name we should either use it in the metric name `${gameName}.pomelo.response_time_ms` or keep the game tag. In terms of datadog costs it's equivalent.